custom/icds_reports/reports/issnip_monthly_register.py

```diff
@@ -371,7 +359,123 @@ def agg_child_health_monthly(self):
                 'age_tranche__in': ['6', '12', '24', '36'],
                 'minority': 'yes'
             }, 'rations_21_plus_distributed')),
+            pre_sc_boys_36_72=Sum(self.filter_by({
+                'caste': 'sc',
+                'gender': 'M',
+                'age_tranche__in': ['48', '60', '72']
+            }, 'rations_21_plus_distributed')),
```

Comment:
These definitions changed from `pse_days_attended` to `rations_21_plus_distributed`. It looks like in agg child health you we don't have a pse_days_attended > 21 column, so you should be able to query child_health_monthly view with these same columns but counting the rows where `pse_days_attended >=21`